demo/universal/native/lib/ServerFunctions.re

```diff
@@ -79,17 +82,23 @@ module Notes = {
   };
   let create =
     switch%platform () {
-    | Server => (
-        (~title, ~content) =>
-          failwith("We don't support Server Functions on server side yet")
-      )
-    | Client => (
-        (~title, ~content) => {
-          // Register the action for the client
-          let action = ReactServerDOMEsbuild.createServerReference(createId);
-          action(. title, content);
-        }
-      )
+    | Server => {
+        Runtime.React.id: Some(createId),
+        call: ((. ~title, ~content) => createHandler(~title, ~content)),
+      }
+    | Client => {
+        Runtime.React.id: Some(createId),
+        call: (
+          (. ~title, ~content) => {
+            let action =
+              ReactServerDOMWebpack.createServerReference(
```

Comment:
any reason to use DOMWebpack here?

Reply:
Ops, it was present on the previous branch and I didn't pay enough attention on it. Fixing